bcl2fastq/index_summary.py

```diff
@@ -100,13 +100,19 @@
 
 
 @app.callback(
-    dep.Output('url', 'pathname'),
-    [dep.Input('run_select', 'value')]
+    [dep.Output('run_select', 'value'),
+     dep.Output('error', 'displayed')],
+    [dep.Input('url', 'pathname')]
 )
-def change_url(dropdown_value):
-    """Allows user to enter Run name in URL which will update dropdown automatically, and the graphs
+def change_url(value):
+    """Allows user to enter Run name in URL which will update dropdown automatically, and the graphs.
+    If User enters any value that's not a valid run an error box will pop up and return user to most recent run
     """
-    return dropdown_value
+
+    if value == "/" or str(value)[1:] not in all_runsog:
```

Comment:
Add a comment explaining why you are getting everything but the first character of a string.